### PR TITLE
uptime: add page

### PR DESCRIPTION
**Pull request by igorshubovych**

### CODE REVIEW

**Comment by ostera** [comment]
:+1:

**Comment by felixonmars** [comment]
:+1: :smile_cat:

File: pages/common/uptime.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# uptime
+
+> Tell how long the system has been running and other information
+
+- Print current time, uptime, logged users amount and other information
```
Comment (waldyrious) [inline pages/common/uptime.md]: "logged users amount" --> "number of logged-in users"?

Comment (igorshubovych) [inline pages/common/uptime.md]: As it is merged do a new pull-request please. My English is bad (sorry for this), hence I am always doing pull requests to hear everybody's opinions.

Comment (waldyrious) [inline pages/common/uptime.md]: Sure, just confirming that that was the original intent.

**Pull request merged**